README.md

```diff
@@ -1,44 +1,375 @@
 # VoiceWP
 
+Create Alexa Skills through WordPress
+
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/voicewp/Lobby)
+
 VoiceWP is a WordPress plugin that integrates with Amazon Alexa to create and enable the creation of Alexa skills.
 
-It allows you to use WordPress as an endpoint.
+## How it Works
+
+VoiceWP creates REST endpoints using the WordPress REST API. These endpoints handle all of the logic of your Alexa skills. When a user interacts with your skill, it sends a Request to an endpoint, which is processed by this plugin, and sends the Response back to the user.
```

Comment:
the plugin immediately turns them into objects of those classes, so to keep the consistency of the class names i went with capitalized